### PR TITLE
FIXes some fixes

### PR DESCRIPTION
**Pull request by ChD1**
**Checklist**

- [ ] Closing issues: #issue
- [ ] Mark this if you consider it ready to merge
- [ ] I've added tests (optional)
- [ ] I wrote some lines in the [radare2book](https://github.com/radareorg/radare2book)

**Description**

### CODE REVIEW

File: libr/core/panels.c [hunk 1]
```diff
@@ -363,6 +367,7 @@ static bool __check_if_mouse_x_on_edge(RCore *core, int x, int y) {
 	int i = 0;
 	for (; i < panels->n_panels; i++) {
 		RPanel *panel = __get_panel (panels, i);
+		if (!panel) continue;
```
Comment (trufae) [inline libr/core/panels.c:370]: please follow the coding style guide. there are no single line ifs. always add braces and a newline

File: libr/core/panels.c [hunk 2]
```diff
@@ -972,6 +989,7 @@ static void __update_help(RCore *core, RPanels *ps) {
 	int i;
 	for (i = 0; i < ps->n_panels; i++) {
 		RPanel *p = __get_panel (ps, i);
+		if (!p) continue;
 		if (!strncmp (p->model->cmd, help, strlen (help))) {
```
Comment (trufae) [inline libr/core/panels.c:993]: if (p && !strcmp ...) looks better to me

File: libr/core/panels.c [hunk 3]
```diff
@@ -958,6 +974,7 @@ static void __adjust_side_panels(RCore *core) {
 	RPanels *panels = core->panels;
 	for (i = 0; i < panels->n_panels; i++) {
 		RPanel *p = __get_panel (panels, i);
+		if (!p) continue;
 		if (p->view->pos.x == 0) {
```
Comment (trufae) [inline libr/core/panels.c:978]: same

**Review by trufae — CHANGES_REQUESTED** [review]
.

File: libr/core/panels.c (hunk 3 above)
Comment (ChD1) [reply libr/core/panels.c]: done

**Review by trufae — APPROVED** [review]

**Pull request merged**